main/tests/server/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperationTests.java

```diff
@@ -337,7 +337,7 @@ public void testExponentialRetries() throws Exception {
             }
 
             // Use 503 Server Unavailable with no Retry-After header this time
-            for (int i = 0; i < 5; i++) {
+            for (int i = 0; i < 4; i++) {
```

Comment:
I don't think this should be changed, and changing this will cause inconsistency with the comment below in the code.

```java
...
// 1st row fails after 4 tries (3 retries), 2nd row tries twice and gets value, 3rd row is hard error
...
```

Reply:
I changed it because I thought that the previous version was inconsistent with the comment actually. This adds four 503 requests: 4 tries (3 retries). The previous code added 5 of them - do you understand why?

Reply:
Seems you ignored `2nd row tries twice and gets value`. Besides the four 503 responses consumed by the first row, the second row is also supposed to get a 503 response on the first try,

Reply:
That's why there's another 200ms at the end of the following comment:

```
// 6 requests (4 retries 200, 400, 800, 200 msec) + final response
```

Reply:
Oops, thanks indeed! sorry about that.